src/app/components/form/form/form.component.html

```diff
@@ -0,0 +1,109 @@
+
+<form #myForm="ngForm" (ngSubmit)="onSubmit(myForm)">
+    <div>
+        <!-- Form check -->
+        @if (myForm.invalid) {
+        <em class="msgColorAlert">Formulaire à completer</em>
+        } @else if (!myForm.submitted) {
+        <em class="msgColorValid">completé,</em><em class="msgColorAlert"> à envoyer</em>
+        } @else {
+        <em class="msgColorValid"> Form submitted : {{myForm.submitted}}, valid : {{myForm.valid}}. invalid :
```

Comment:
use french for content, and english for coding. And display "invalid " and "valid" stuffs only if they're some